pkg/query-service/app/logs/v3/query_builder.go

```diff
@@ -330,9 +320,20 @@ func orderBy(items []v3.OrderBy, tags []string) string {
 	for _, item := range items {
 		if item.ColumnName == constants.SigNozOrderByValue {
 			orderBy = append(orderBy, fmt.Sprintf("value %s", item.Order))
+			addedToGroupBy[item.ColumnName] = true
+		}
+	}
+
+	// add the remaining items
+	for _, item := range items {
+		// since these are not present in tags we will have to select them correctly
+		if !addedToGroupBy[item.ColumnName] {
+			attr := v3.AttributeKey{Key: item.ColumnName, DataType: item.DataType, Type: item.Type, IsColumn: item.IsColumn}
+			name, _ := getClickhouseColumnName(attr)
+			orderBy = append(orderBy, fmt.Sprintf("%s %s", name, item.Order))
```

Comment:
>since these are not present in tags we will have to select them correctly

Are these part of the select labels? (i.e a label need to be in SELECT ... before we can use them in orderby)

Reply:
As we have decided to return all fields instead of user selecting some fields in the api response. So orderby doesn't need to check select.